velox/docs/functions/spark/string.rst

```diff
@@ -23,6 +23,21 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT contains('Spark SQL', null); -- NULL
         SELECT contains(x'537061726b2053514c', x'537061726b'); -- true
 
+.. spark:function:: conv(number, fromBase, toBase) -> varchar
+
+    Converts ``number`` represented as a string from ``fromBase`` to ``toBase``.
+    ``fromBase`` must be a value inclusively between 2 and 36. ``toBase`` must be a value
+    inclusively between 2 and 36 or between -36 and -2. Negative ``toBase`` means output
+    will be a signed number and positive ``toBase`` means it will be an unsigned number.
+    If ``number`` is empty or given base is invalid, returns NULL.
```

Comment:
> given base is invalid

What does that mean? Can you provide an example?

typos: Returns NULL if ...

Reply:
It means ``fromBase`` is not a number between 2 and 36, or `toBase` is not a number between 2 and 36, or between -36 and -2. For example, fromBase = 37.
I will refine this part to avoid misunderstanding.